src/tests/components/DeleteModal.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { DeleteModal } from '../../components/Articles/DeleteModal';
+
+describe('DeleteModal Component', () => {
+  let wrapper;
+  const confirmDelete = true;
+  const deleteArticle = jest.fn();
+  const slug = 'react-redux';
+
+  const getEvent = (name = '', value = '') => ({
+    preventDefault: jest.fn(),
+    target: {
+      name,
+      value,
+    },
+  });
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <DeleteModal
+        slug={slug}
+        confirmDelete={confirmDelete}
+        deleteArticle={deleteArticle}
+      />,
+    );
+  });
+
+  it('should render the Articles Component correctly', () => {
```

Comment:
Is this an Articles component?

Reply:
Thanks for pointing that out I have changed it.